e2e.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash -e
 
-python manage.py behave --settings=sheetstorm.settings.testing --no-capture
+coverage run --source='.' --rcfile=coverage-config manage.py behave --settings=sheetstorm.settings.testing --no-capture
+coverage report
+rm .coverage
```

Comment:
I'm not sure but we in all shell files add newline at the end of file. Please add it also here :)